pkg/apm/applications.go

```diff
@@ -88,6 +73,54 @@ func (apm *APM) DeleteApplication(applicationID int) (*Application, error) {
 	return &response.Application, nil
 }
 
+// MetricNames is used to retrieve a list of known metrics and their value names for the given resource.
+//
+// https://rpm.newrelic.com/api/explore/applications/metric_names
```

Comment:
Nice. We've only done this in a few places so far. I like the idea of a documentation pattern where we include links to docs if a method maps directly to an endpoint/doc like this. 😎

Reply:
Let's hope the link for the docs doesn't change :)